tests/vmcheck/multitest.py

```diff
@@ -102,10 +102,12 @@ def flush(self):
         rc = self._p.wait()
 
         # just merge the two files
-        with open("vmcheck/%s.out" % self.test) as f:
-            with open("vmcheck/%s.log" % self.test, 'a') as j:
-                j.write(f.read())
-        os.remove("vmcheck/%s.out" % self.test)
+        outfile = "vmcheck/{}.out".format(self.test)
+        if os.path.isfile(outfile):
```

Comment:
Not a hack! I think that makes sense. If `test.sh` failed even before writing to the log file, opening will fail.